frontends/election-manager/src/screens/unconfigured_screen.tsx

```diff
@@ -214,7 +215,7 @@ export function UnconfiguredScreen(): JSX.Element {
   }, [updateStatus]);
 
   useEffect(() => {
-    if (location.pathname !== '/') {
+    if (!areVvsg2AuthFlowsEnabled() && location.pathname !== '/') {
```

Comment:
I know there's a lot of feature-flag based branching in this commit. I just want to minimize interruptions for the NH team this week! I've also created a ticket to eventually remove this feature flag and corresponding old code branches: https://github.com/votingworks/vxsuite/issues/2085